server/src/main/java/io/deephaven/server/grpc_api_app/SessionStateExportObjectUpdateStreamPublisher.java

```diff
@@ -0,0 +1,87 @@
+package io.deephaven.server.grpc_api_app;
+
+import io.deephaven.chunk.WritableChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.table.ColumnDefinition;
+import io.deephaven.engine.table.TableDefinition;
+import io.deephaven.engine.table.impl.perf.QueryPerformanceRecorder;
+import io.deephaven.engine.table.impl.sources.ArrayBackedColumnSource;
+import io.deephaven.proto.backplane.grpc.ExportNotification;
+import io.deephaven.qst.type.Type;
+import io.deephaven.stream.StreamChunkUtils;
+import io.deephaven.stream.StreamConsumer;
+import io.deephaven.stream.StreamPublisher;
+import io.deephaven.time.DateTimeUtils;
+import io.deephaven.util.BooleanUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.Objects;
+
+final class SessionStateExportObjectUpdateStreamPublisher implements StreamPublisher {
+
+    private static final TableDefinition DEFINITION = TableDefinition.of(
+            ColumnDefinition.ofTime("Time"),
+            ColumnDefinition.ofLong("EvaluationNumber"),
+            ColumnDefinition.ofString("SessionId"),
+            ColumnDefinition.ofString("ExportId"),
+            ColumnDefinition.ofBoolean("IsNonExport"),
+            ColumnDefinition.ofString("State"),
+            ColumnDefinition.ofBoolean("IsLive"),
+            ColumnDefinition.ofString("Description"),
+            ColumnDefinition.of("RemainingDependencies", Type.find(String[].class)));
```

Comment:
I picked string because all of the error messages use this hex-style id that is convenient for humans.

I'm willing to discuss; will let Colin add his opinion(s).